### ISSUE TITLE
Review test descriptions and update contributor guide

### ISSUE THREAD

**Issue by y4izus** [Qiskit/qiskit.org#522]
It’s a good practice to “describe” the test steps and assertions, just like you do when you write a checklist. We are using [jest](https://jestjs.io/), so we could do it with “describes” and “its” this way:

```
describe('filteredEvents', () => {
    it('gets future events by default', () => {
      ...
    })
})
```

Notice that you can read `describe` + `it` in a single phrase: "filteredEvents gets future events by default"

We can see [more examples](https://github.com/facebook/jest/tree/master/examples) in the jest repository

We should unify the code in order to use the test the same way everywhere. And also, we should update the [contributor guidelines](https://github.com/Qiskit/qiskit.org/blob/master/CONTRIBUTING.md) to inform the contributors about this way of doing tests

**Comment by delapuente** [comment]
Agree, got it changed in #519 if it is not too noisy for the PR.

**Comment by delapuente** [comment]
Closed by #519
